.github/workflows/e2e-tests.yml

```diff
@@ -0,0 +1,125 @@
+name: E2E Tests for PR
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+    paths-ignore:
+      - "docs/**"
+      - "**.md"
+      - ".circleci/**"
+
+jobs:
+
+  build:
+    if: github.event.pull_request.draft == false
```

Comment:
[❤️](https://emojipedia.org/red-heart/)

Reply:
Isn't this a duplicate of the

```yml
on:
  pull_request:
    types: [ready_for_review]
```

?

Reply:
Ooooooooh, no no no... I see what you did here :)

I was convinced that the default functionality for `ready_for_review` was basically the same as for `opened` IF the PR wasn't in the draft mode. It feels silly that `ready_for_review` doesn't trigger the workflow immediately after it's opened.

Reply:
Yeah, that was my first attempt and it didn't work well. Hence, the circus dance with various `pull_request` triggers.